nodes/BoolOp.py

```diff
@@ -4,18 +4,34 @@
 A boolean operation, 'or' or 'and'.
 
 Attributes:
-    - values  (Expr)
-        - The values involved.
-    - op      (Or | And)
+    - values  ([Expr])
+        - A list of values involved
```

Comment:
Say here "A list of the argument expressions" to be a bit more specific

Reply:
I have made changes accordingly.